app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt

```diff
@@ -100,7 +98,7 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
      * Used to cancel background->UI threads: we attach them as children to this job
      * and cancel this job at the end of the UI lifecycle, cancelling the children.
      */
-    var uiLifecycleCancelJob: Job
+    var uiLifecycleCancelJob: Job = Job()
```

Comment:
If this is destroyed in `onDestroyView`, it should be initialized in `onCreateView`, right?